api/control.yaml

```diff
@@ -0,0 +1,175 @@
+          application/json:
+            schema:
+              $ref: './all.yaml#/components/schemas/UserResource'
+      responses:
+        '200':
+          description: The new user was successfully created.
+          content:
+            application/json:
+              schema:
+                $ref: './all.yaml#/components/schemas/User'
+    get:
+      tags:
+        - Users
+      parameters:
+        - name: filter
+          in: query
+          description: Query by which the results have to be filtered.
+          schema:
+            type: string
+          required: false
+        - name: startIndex
+          in: query
+          description: Specifies the index of the first result. First item is number 1.
+          schema:
+            type: integer
+            format: int32
+          required: false
+        - name: count
+          in: query
+          description: Desired number of results per page.
+          schema:
+            type: integer
+            format: int32
+          required: false
+      operationId: listUsers
+      summary: List users
+      description: |
+        Gets details for all the users.
+      responses:
+        '200':
+          description: The user list was successfully retrieved.
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  $ref: './all.yaml#/components/schemas/UserResource'
+  /scim2/users/{id}:
+    parameters:
+      - name: id
+        in: path
+        description: The id of the user.
+        required: true
+        schema:
+          type: string
+    get:
+      tags:
+        - Users
+      operationId: getUser
+      summary: Get a user
+      description: |
+        Gets the specified user.
+      responses:
+        '200':
+          description: The user was successfully retrieved.
+          content:
+            application/json:
+              schema:
+                $ref: './all.yaml#/components/schemas/UserResource'
+    put:
+      tags:
+        - Users
+      operationId: updateUser
+      summary: Update a user
+      description: |
+        Updates the user that matches the supplied id.
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: './all.yaml#/components/schemas/UserResource'
+      responses:
+        '200':
+          description: The user was successfully updated.
+          content:
+            application/json:
+              schema:
+                $ref: './all.yaml#/components/schemas/UserResource'
+    delete:
+      tags:
+        - Users
+      operationId: deleteUser
+      summary: Delete a user
+      description: |
+        Deletes the user that matches the supplied id.
+      responses:
+        '200':
+          description: The user was successfully deleted.
+          content:
+            application/json:
+              schema: {}
+  /permissions/{resource_type}/{name}:
```

Comment:
Maybe a little odd, but I think this should be in `all.yaml` since we're duplicating an endpoint that exists in DBX UC.

Reply:
... and it might already be in `all.yaml` ?